frontend/packages/dev-console/src/components/topology/componentUtils.ts

```diff
@@ -215,8 +241,12 @@ export {
   nodeDragSourceSpec,
   nodeDropTargetSpec,
   graphWorkloadDropTargetSpec,
-  groupWorkoadDropTargetSpec,
+  groupWorkloadDropTargetSpec,
+  graphEventSourceDropTargetSpec,
   edgeDragSourceSpec,
   createConnectorCallback,
   removeConnectorCallback,
+  MOVE_CONNECTOR_DROP_TYPE,
+  MOVE_EV_SRC_CONNECTOR_DROP_TYPE,
+  MOVE_CONNECTOR_OPERATION,
```

Comment:
I believe you can remove all references to `MOVE_CONNECTOR_OPERATION` now.

Reply:
have removed `MOVE_CONNECTOR_OPERATION` and other type is used  as param to `edgeDragSourceSpec` currently